src/components/application_manager/src/commands/command_request_impl.cc

```diff
@@ -37,10 +37,44 @@
 #include "application_manager/application_manager.h"
 #include "application_manager/message_helper.h"
 #include "smart_objects/smart_object.h"
+
 namespace application_manager {
 
 namespace commands {
 
+std::string MergeInfos(const std::string& first, const std::string& second) {
+  return first + ((!first.empty() && !second.empty()) ? ", " : "") + second;
+}
+
+std::string MergeInfos(const std::string& first,
+                       const std::string& second,
+                       const std::string& third) {
+  std::string result  = MergeInfos(first, second);
+  return MergeInfos(result, third);
+}
+
+void CreateInfoForUnsupportedResult(std::string& info,
+                                   HmiInterfaces::InterfaceID interface) {
+    switch (interface) {
+    case(HmiInterfaces::InterfaceID::HMI_INTERFACE_VR):{
+        info = "VR is not supported by system";
+        break;
+    } case(HmiInterfaces::InterfaceID::HMI_INTERFACE_TTS): {
+        info = "TTS is not supported by system";
+        break;
+    } case(HmiInterfaces::InterfaceID::HMI_INTERFACE_UI): {
+        info = "UI is not supported by system";
+        break;
+    } case(HmiInterfaces::InterfaceID::HMI_INTERFACE_Navigation): {
+        info = "Navi is not supported by system";
+        break;
+    }
+    default:
+    break;
```

Comment:
@dtrunov,
Creation little stack object is not a big issue for logging some unexpected behavior.